npm/vue/examples/code-coverage/cypress/integration/spec.js

```diff
@@ -0,0 +1 @@
+// this file has to stay to avoid creating examples
```

Comment:
You can also just add a `.keep` file - this seems pretty common. Like this: https://github.com/cypress-io/cypress/tree/develop/npm/react/examples/nextjs-webpack-5/cypress/fixtures

But both fine.